src/main/java/pl/edu/agh/ki/io/db/FriendshipRepository.java

```diff
@@ -0,0 +1,15 @@
+package pl.edu.agh.ki.io.db;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import pl.edu.agh.ki.io.models.Friendship;
+import pl.edu.agh.ki.io.models.User;
+
+@Repository
+public interface FriendshipRepository extends JpaRepository<Friendship, Long> {
+
+    Friendship findByRequesterAndAddressee(User requester, User addressee);
```

Comment:
chyba wszędzie używamy Optional w repozytoriach i wtedy nie było by porównywania z nullem tylko .isPresent(). Jak na mnie trochę ładniejsze jest, ale up to you